README.md

```diff
@@ -79,6 +80,7 @@ See [CONTRIBUTING.md](docs/CONTRIBUTING.md).
 
 - [Brave Search](styles/brave-search)
 - [Ecosia](styles/ecosia)
+- [Google](styles/google)
```

Comment:
This should be autogenerated (right?) so not sure why there are changes here.

Reply:
Idk, last PR I made I added it manually